platform/windows/os_windows.cpp

```diff
@@ -386,6 +387,10 @@ LRESULT OS_Windows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
 			// Don't calculate relative mouse movement if we don't have focus in CAPTURED mode.
 			if (!window_has_focus && mouse_mode == MOUSE_MODE_CAPTURED)
 				break;
+			if(ignore_next_mousemove) {
```

Comment:
Wrong style (check two lines above for a hint ;)).

https://travis-ci.org/godotengine/godot/jobs/314130965

Reply:
Gneeeeh my clang-format doesn't work, I have to update LLVM...